app/script/auth/page/ConversationJoin.jsx

```diff
@@ -91,15 +103,6 @@ class ConversationJoin extends Component {
 
   componentWillReceiveProps = nextProps => this.readAndUpdateParamsFromUrl(nextProps);
 
-  openWebapp = params => {
-    const link = document.createElement('a');
-    link.href = pathWithParams(EXTERNAL_ROUTE.LOGIN, params);
-    document.body.appendChild(link); // workaround for Firefox
```

Comment:
Do we no longer need the workaround for Firefox? We did not remove it from https://github.com/wireapp/wire-webapp/pull/2896/files#diff-5faba9f0446a38fd86c2ec6241b9f713R67

Reply:
The workaround is no longer necessary in this case, as we no longer redirect to a different page.